README.md

```diff
@@ -44,28 +51,55 @@ command line like this:
 ./normalizer < sample.csv > output.csv
 ```
 
+### Web Browser (If you want us to run your code from a browser):
+
+Please write a static webpage that normalizes and displays the data contained in
+`sample.csv`. We have provided a function that will return the contents of this
+file as plain text, which you can either include directly in the browser, or
+import to your compiled JavaScript. If you would like to import it, you can
+modify `getSampleData.js` to export the `getSampleData` function. For example:
```

Comment:
Meta comment that may have already been discussed and so may not be appropriate for the PR.  When we ask folks to handle stdin/stdout it's to see if they can handle something on the backend that may not be as straightforward as opening a file.  I feel like in the browser, the equivalent is having a form that accepts a file.

We don't provide anything similar to sample code for the command line tool.

Reply:
Another option would be to provide an endpoint for them to hit that returns the CSV.  We can ask that the input URL be an input to the form.  We can provide an endpoint for the sample CSV and then have a different endpoint for our actual test data